index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/e746991a9e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Ibute_uchenna-portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <div class="logo">
+            <a class="lo" href="index.html">IBUTE~UCHENNA</a>
+        </div>
+        <nav class="menu">
+            <i class="fa-solid fa-bars"></i>
+        </nav>
+    </header>
+    <section class="section1">
+        <div>
+            <h1 class="head">Hello <br> I'm Ibute Uchenna</h1>
+        </div>
+        <div>
+            <h2 class="title">I'm a Software Developer</h2>
+        </div>
+        <div class="para">
+            <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="icons">
+            <a href="#"><i class="fa-brands fa-linkedin fas"></i></a>
+            <a href="#"><i class="fa-brands fa-twitter fas"></i></a>
+            <a href="#"><i class="fa-brands fa-github fas"></i></a>
+            <a href="#"><i class="fa-brands fa-facebook fas"></i></a>
+        </div>
+        <div class="bg-design">
+            <img class="imga" src="img/v7.png">
+            <img class="imgb" src="img/v8.png">
+            <img class="imgc" src="img/v9.png">
+            <img class="imge" src="img/v11.png">
+            <img class="imgf" src="img/V12 (2).png">
+            <img class="imgf1" src="img/V12 (2).png">
+            <img class="imgf2" src="img/V12 (2).png">
+            <img class="imgf3" src="img/V12 (2).png">
+            <img class="imgg" src="img/V13.png">
+            <img class="imgh" src="img/V13.png">
+        </div>
```

Comment:
[ OPTIONAL ] Please make sure you use the `ul` and `li` tags to list the items in your project. You can read more about it [here](https://www.w3schools.com/html/html_lists.asp) :pray: